pkg/pillar/types/locationconsts.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017 Zededa, Inc.
```

Comment:
Still 2017?

Reply:
Not sure what it is meant to be.. I was just moving the lines to a new file.. so not sure if I need to use the older date or the newer date